dashboard/test/controllers/api_controller_test.rb

```diff
@@ -1270,6 +1270,61 @@ class ApiControllerTest < ActionController::TestCase
     assert_equal 1, data['students'].length
   end
 
+  test "should get paginated section level progress" do
```

Comment:
these are very similar to the older API used for the current progress tab, but they return data in a different format.